.circleci/bin/needsmodel

```diff
@@ -5,17 +5,21 @@ set -o nounset
 set -o pipefail
 set -o noclobber
 
-versionchanged ()
-{
-  git diff "$(git merge-base main HEAD)"..HEAD -- search-config.toml \
-    | grep -e +version -e -version
-}
-
 needsmodel ()
 {
-  if [[ -z "$(versionchanged)" ]]; then
+  versions_unsorted=$(git --no-pager diff main search-config.toml | grep version | sed 's/^[+|-]//g')
```

Comment:
I'm relying on the fact that lexical sorting of diffed versions results in a different ordering when we're reverting to an older release here

```diff
❯ git --no-pager diff main search-config.toml
diff --git a/search-config.toml b/search-config.toml
index 3a29b7d..361a5c7 100644
--- a/search-config.toml
+++ b/search-config.toml
@@ -1,5 +1,5 @@
 name = "fpo-search-model-generator"
-version = "1.3.0"
+version = "1.3.1"
 learning_rate = 0.0011
 max_epochs = 4
 model_batch_size = 1000
```